src/stamps/entries.js

```diff
@@ -32,13 +32,14 @@ const stamp = stampit()
 .methods({
   /**
    * Get all the content entries, based on the given parameters.
-   * **DO NOT** use several of id, alias and typeId at the same time - behaviour would be ungaranteed.
+   * **DO NOT** use several of id, alias, typeId and typeAlias at the same time - behaviour would be ungaranteed.
    * @param {object} [params] a parameters object
    * @param {boolean} [params.preview] when true, get the preview version
    * @param {string|date} [params.at] when given, get the version at the given time
    * @param {array} [params.id] an array of entry ids (strings)
    * @param {array} [params.alias] an array of entry aliases (strings)
    * @param {string} [params.typeId] only return entries matching this type id
```

Comment:
the apiary doc is a bit misleading..
the title says Get Entries by Type ID.. but the description says a comma separated list of type ids..
Also there is a change needed for Get Entries by Type Alias.. in this case the title is correct as we have not yet implemented multiple aliases.. but the description says we support list of aliases..
![screen shot 2016-10-24 at 11 09 31](https://cloud.githubusercontent.com/assets/11408024/19639912/b9c8f9a4-99da-11e6-8339-e36ce9a870f9.png)

I ll create a ticket to get it fixed..